source/f7_testing/main.c

```diff
@@ -46,4 +46,4 @@ void HardFault_Handler()
     {
         __asm__("nop");
     }
-}
```

Comment:
Try not to commit non-changes like this in the future. Small nit but other than that approved.

Reply:
I tried to fix it and seem to have made it worse lol. Different text editors seem to treat files differently in terms of adding a newline at the end.

Reply:
> I tried to fix it and seem to have made it worse lol. Different text editors seem to treat files differently in terms of adding a newline at the end.

I removed the other commit, I am unsure why my editor is doing this and I will try to fix it for the future. Would you like it fixed before merging or does it not matter?